2022/pt-br/Days/day05.md

```diff
@@ -43,43 +43,42 @@ Uma vez implantado, vamos operá-lo. E operá-lo pode envolver algo como você c
 
 ## Monitoramento
 
-All of the above parts lead to the final step because you need to have monitoring, especially around operational issues auto-scaling troubleshooting like you don't know
-there's a problem if you don't have monitoring in place to tell you that there's a problem so some of the things you might build monitoring for are memory utilization CPU utilization disk space, API endpoint, response time, how quickly that endpoint is responding and a big part of that as well is logs. Logs give developers the ability to see what is happening without having to access production systems.
+Todas as partes acima levam à etapa final porque você precisa ter monitoramento, especialmente em torno de questões operacionais, solução de problemas de escalonamento automático, como se você não soubesse que há um problema se não tiver monitoramento instalado para informar que há um problema portanto, algumas das coisas para as quais você pode criar monitoramento são utilização de memória, utilização de espaço em disco, utilização da CPU, API endpoint, tempo de resposta, a rapidez com que esse endpoint está respondendo e uma grande parte disso também são logs. Os logs oferecem aos desenvolvedores a capacidade de ver o que está acontecendo sem precisar acessar os sistemas de produção.
 
-## Rinse & Repeat
+## Limpe & Repita
 
-Once that's in place you go right back to the beginning to the planning stage and go through the whole thing again
+Uma vez implementado, você volta ao início, ao estágio de planejamento, e repassa tudo novamente.
 
-## Continuous
+## Contínua
 
-Many tools help us achieve the above continuous process, all this code and the ultimate goal of being completely automated, cloud infrastructure or any environment is often described as Continuous Integration/ Continuous Delivery/Continous Deployment or “CI/CD” for short. We will spend a whole week on CI/CD later on in the 90 Days with some examples and walkthroughs to grasp the fundamentals.
+Muitas ferramentas nos ajudam a alcançar o processo contínuo acima, todo esse código e o objetivo final de ser completamente automatizado, a infraestrutura em nuvem ou qualquer ambiente é frequentemente descrito como Integração Contínua/Entrega Contínua/Implantação Contínua ou “CI/CD” para abreviar. Passaremos uma semana inteira sobre CI/CD mais tarde nos 90 dias com alguns exemplos e orientações para compreender os fundamentos.
 
-### Continuous Delivery
+### Entrega Contínua
 
-Continuous Delivery = Plan > Code > Build > Test
+Entrega Contínua = Planejar > Codar > Construir > Testar
```

Comment:
I would suggest using Nouns instead of Verbs:
"Entrega Contínua = Planejamento > Codificação > Construção > Teste"